src/firebase/firebase.ts

```diff
@@ -189,6 +189,8 @@ export function* initializeCloudMessaging(app: ReactNativeFirebase.Module, addre
   const isEmulator = yield* call([DeviceInfo, 'isEmulator'])
   // Emulators can't handle fcm tokens and calling getToken on them will throw an error
   if (!isEmulator) {
+    yield* call([CleverTap, 'registerForPush'])
+    yield* call([app.messaging(), 'registerDeviceForRemoteMessages'])
```

Comment:
i'd like to put this line back @jh2oman if that's okay with you - i'll test this in the nightly build tomorrow to see if it causes issues, but without this line i could not get the push notifications working locally. i looked into whether it could be a signing issue (the only difference i can think of) but i cannot see any reason why it would be :/

Reply:
There were some steps for testing pushes in https://github.com/valora-inc/wallet/pull/2012, that may or may not be useful.

Reply:
We can see how the (nightly) build goes. I'm fine with merging. This line was causing issues when upgrading segment versions before so let's make sure we are also seeing all analytics events on the (nightly) build correctly

Reply:
ty! i'll see how it goes on the nightly, and tweak if it does not work as expected